### ISSUE TITLE
Cucumber coloured output.

### ISSUE THREAD

**Issue by carlosbaraza** [moxley/atom-ruby-test#32]
I am using an old version of ruby (`REE 1.8.7`) and cucumber (`0.10.5`). In order to get coloured output, I had to set the cucumber commands like follows:

```
'ruby-test':
    'cucumberSingleCommand': 'cucumber --color {relative_path}:{line_number}'
    'cucumberFileCommand': 'cucumber --color {relative_path}'
    'cucumberAllCommand': 'cucumber --color features'
```

I am not sure if including this option hardcoded could cause some issues with newer versions. Otherwise, at least we could include this information in the documentation, so users like me could find this information easily and have a better experience with the plugin.

Congratulations for this nice package,
Carlos.

**Comment by dmeehan1968** [comment]
Even with cucumber v1.3.18, you still need to use to --color option to see coloured output.  There is a commit that references the ansi-to-html plugin, but this plugin does not appear to exist within the plugin repository.

**Comment by moxley** [comment]
Now available in version 0.9.3. Thanks for your contribution!
